stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -327,6 +327,11 @@ public void request(int count) {
     public void setMessageCompression(boolean enable) {
       call.setMessageCompression(enable);
     }
+
+    @Override
+    public void cancel() {
```

Comment:
`String` makes sense, but why `Throwable`? Why should I have to provide an exception class when I am intentionally canceling the stream? i.e. not an exceptional situation.